src/Components/categories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Categories = () => (
+  <div className="navContainer">
+    <h1 className="heading">Welcome to the Categories Page</h1>
+  </div>
```

Comment:
- [ ] Following the project requirements, the Categories component should display only a button with the text "Check status".